conda/cli/install.py

```diff
@@ -419,8 +418,6 @@ def install(args, parser, command="install"):
     if (isinstall or isremove) and args.update_modifier == NULL:
         update_modifier = UpdateModifier.FREEZE_INSTALLED
     deps_modifier = context.deps_modifier
-    if isupdate:
-        deps_modifier = context.deps_modifier or DepsModifier.UPDATE_SPECS
```

Comment:
`DepsModifier` does not have an attribute called `UPDATE_SPECS` https://github.com/conda/conda/blob/main/conda/base/constants.py#L201